### ISSUE TITLE
[A11yMAS][Accessibility][Keyboard][MAS2.4.3][Eclipse][Wipro]: Keyboard focus order is illogical in "Install Details" Window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1351]
**Repro Steps:**

1. Open and Start Eclipse.
2. Press "Alt+H" to open Help menu.
3. Navigate and select Install new software Menu item under Help menu.
4. In Available Software dialog , navigate to Work with text edit box and provide http://dl.microsoft.com/eclipse/ link and hit enter
5. Then select "Azure Toolkit" checkbox and Tab  up to "Contact all update sites during install to find required software" checkbox then press tab to "Next" button and hit enter.
6. In Install details dialog, tab up to "Details" and press tab then observe the keyboard focus order

**Expected Result:**
Keyboard focus should be Details->Help->Back->Next->Cancel

**Actual Result:**
Two tab Loss of focus is observed while navigating from Details->Back button

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version:** Version 1709(OS Build 16299.192)

![keyboard focus is illogical in install window](https://user-images.githubusercontent.com/22956130/35389646-8a289726-01fe-11e8-9270-74c9caecaaad.jpg)

**Comment by Eskibear** [comment]
It is determined by Eclipse IDE. We cannot fix it.

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=534670

**Comment by Eskibear** [comment]
* **Importance of experience**: somewhat important. It's about the one-time installation.
* **Quality of experience**: not impacted. Simple workaround is available, users can use **arrow keys** to navigate after it's focusing on "Back". Or users can simply install the plugin from **Help->Marketplace** instead of **Help->Install New Software**.  
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749828
